### PR TITLE
Aot demo2

### PR DESCRIPTION
**Pull request by mlaval**
Follows #794, only the second commit is to be reviewed.

With AoT, loading the demo site goes down from 2.5s to 0.9s (in Chrome, local server).

### CODE REVIEW

**Comment by mlaval** [comment]
Updated to use `@ngtools/webpack`, much better!

File: src/modal/modal.spec.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {NgbModalModule, NgbModal, NgbActiveModal, NgbModalRef} from './modal.mod
 const NOOP = () => {};
 
 @Injectable()
-class SpyService {
+export class SpyService {
```
Comment (pkozlowski-opensource) [inline src/modal/modal.spec.ts]: Curious as why this change was necessary?

**Comment by pkozlowski-opensource** [comment]
@mlaval I've merged the WebPack 2.0 but now this one needs rebasing (just drop one commit) :-/ 
Also I've left one minor comment. Can't wait to land this one :-)

File: src/modal/modal.spec.ts (hunk 1 above)
Comment (mlaval) [inline src/modal/modal.spec.ts]: At some point, the Angular compiler was yelling at me for this.
But it was a bad configuration where tests were compiled, not needed anymore.

**Comment by mlaval** [comment]
PR rebased and updated
